components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Deploy/IoTHubPublisherDeployment.cs

```diff
@@ -40,7 +40,7 @@ public sealed class IoTHubPublisherDeployment : IHostProcess {
             await _service.CreateOrUpdateConfigurationAsync(new ConfigurationModel {
                 Id = "__default-opcpublisher",
                 Content = new ConfigurationContentModel {
-                    ModulesContent = CreateLayeredDeployment(true)
+                    ModulesContent = CreateLayeredDeployment(true, false)
```

Comment:
>     ModulesContent = CreateLayeredDeployment(true, false) [](start = 15, length = 58)

Used for testing only, please move to e2e-tests/IoTPlatform-E2E-Tests solution, we don't want to modify the product source code.

Reply:
moved